src/show_dialog.cpp

```diff
@@ -201,7 +201,7 @@ dialog_frame::dimension_measurements dialog_frame::layout(int x, int y, int w, i
 	}
 
 	y -= dim_.title.h;
-	w = std::max<int>(w,std::max<int>(int(dim_.title.w),int(buttons_width)));
+	w = std::max<int>(w,std::max<int>(dim_.title.w,int(buttons_width)));
```

Comment:
I've found that the compiler usually _can't_ deduce the template argument for `std::max` and `std::min`, though this may be one of those cases where it does work. Personally I prefer to explicitly specify the template argument rather than using static_cast, when one is necessary.

I think technically `int(buttons_width)` is not a C-style cast but rather a constructor-style cast. Does it actually work in C? Anyway, that's not actually relevant.